util.py

```diff
@@ -167,7 +156,7 @@ def is_yaml_file(path: CliHints.existing_file()):
 def parse_yaml_file(path: CliHints.existing_file(), as_snd=True):
```

Comment:
1. add comment indicating the parameter (as_snd) should be removed (OR: remove it)
2. just return yaml.load(f) (no if/else)